test/source/tests/gmail.ts

```diff
@@ -301,7 +320,9 @@ export const defineGmailTests = (testVariant: TestVariant, testWithBrowser: Test
     );
 
     // convo-sensitive, draft-sensitive test
-    // skipped temporarily as per https://github.com/FlowCrypt/flowcrypt-browser/issues/5934#issuecomment-2664926769, originally uses test.serial
+    // Couldn't figure out why pageHasSecureDraft can't find correct iframe
+    // Need to fix later
+    // https://flowcrypt.semaphoreci.com/jobs/0106da6d-46f5-44d3-9ebd-3421584220a0
     test.skip(
       'mail.google.com - secure reply btn, reply draft',
```

Comment:
@sosnovsky Tried to fix this test but couldn't figure out.
SemaphoreCI couldn't be able to find element in compose draft frame in `pageHasSecureDraft`.
Maybe leave it for future fix?
Or you have other why this happens?